src/middlewared/middlewared/alembic/versions/22.12/2022-09-21_14-35_add_command_line_args.py

```diff
@@ -0,0 +1,31 @@
+"""Add command line args
+
+Revision ID: 7a74a8933a30
+Revises: aef1177c39c5
+Create Date: 2022-09-21 17:47:51.657447+00:00
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = '7a74a8933a30'
+down_revision = 'aef1177c39c5'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    with op.batch_alter_table('vm_vm', schema=None) as batch_op:
+        batch_op.add_column(sa.Column('command_line_args', sa.TEXT(), nullable=False))
```

Comment:
Adding NOT NULL column without a server_default will fail if there already are rows in the table